server/routes/Profile.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+    getProfile,
+    createProfile,
+    updateProfile,
+} = require("../controllers/Profile");
+
+router.route("/").get(protect, getProfile);
+
+router.route("/").post(protect, createProfile);
+
+router.route("/:id").patch(protect, updateProfile);

```

Comment:
```suggestion
router.route("/").patch(protect, updateProfile);

```

I don't see the path params being using in the 'updateProfile' controller